pkg/DESCRIPTION

```diff
@@ -5,10 +5,10 @@ Version: 0.1
 Date: 2013-09-09
 Author: Shivaram Venkataraman
 Maintainer: Shivaram Venkataraman <shivaram@cs.berkeley.edu>
+Imports: methods
```

Comment:
No - I found this out while running a `R CMD check` and reading more about it, it looks like we should be using `Imports` to specify dependencies. http://r-pkgs.had.co.nz/description.html has more details (See under 'Other Dependencies').

To answer the other part of your question we need `methods` to use S4 classes.